### PR TITLE
Add git to host recommendations

### PR DESCRIPTION
**Pull request by chalasr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Prerequisites
 -------------
 
 - [PHP](http://php.net) 7.0+
+- [GIT](https://git-scm.com/) 2.6+
```
Comment (ogizanagi) [inline README.md]: For the record: Why 2.6+?

Comment (chalasr) [reply README.md]: Because it's my current version 😆 😄

Comment (ogizanagi) [reply README.md]: OMG. You're late :trollface: 

```sh
$ git --version
git version 2.9.3
```

https://github.com/git/git/releases

Not sure it worth be as accurate then. Git 2+ should be enough (until someone reports any issue justifying it).

**Comment by nervo** [comment]
Eeeeeeasy, boys :) https://packages.debian.org/jessie/git

**Comment by ogizanagi** [comment]
@nervo : So 2+ is good then 👅 (Anyway required for the diff command to work properly)

**Comment by ogizanagi** [comment]
But actually, it's not a strong requirement right now (only used for diff).

**Comment by chalasr** [comment]
> only used for diff

Hum, `setup` will require it in #85

**Comment by ogizanagi** [comment]
I know 😄 I said **right now**, and we don't have to make it required in order to the setup command to work (as for the deploy role and ansible, it can be considered as a bonus. So a warning could simply advised the user it's required for such cases). We're not relying on git as a core tool to make the project setup as described in https://github.com/manala/manalize/issues/64.

**Comment by chalasr** [comment]
Ok you win

**Comment by chalasr** [comment]
Good to go!

**Comment by chalasr** [comment]
Now it's good to go 😄

**Pull request merged**